alphagov/paas-admin
# Update AWS-SDK packages

What
----

[@aws-sdk/client-cloudwatch-node](https://www.npmjs.com/package/@aws-sdk/client-cloudwatch-node) and [@aws-sdk/client-resource-groups-tagging-api-node](https://www.npmjs.com/package/@aws-sdk/client-resource-groups-tagging-api-node) have been [deprecated](https://github.com/aws/aws-sdk-js-v3/issues/1397) and replaced with [@aws-sdk/client-cloudwatch](https://www.npmjs.com/package/@aws-sdk/client-cloudwatch) and [@aws-sdk/client-resource-groups-tagging-api ](https://www.npmjs.com/package/@aws-sdk/client-resource-groups-tagging-api)respectively

This updated the packages and imports where required

[Modular AWS SDK for JavaScript](https://aws.amazon.com/blogs/developer/modular-aws-sdk-for-javascript-release-candidate/)

How to review
-------------

- deploy to a dev env
- check affected metrics have data

Who can review
---------------

not @kr8n3r

---

🚨⚠️ Please do not merge this pull request via the GitHub UI ⚠️🚨
